src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -65,45 +80,57 @@
  * 4. Implement the following static method in the concrete class:
  *
  *     public static void main(String[] argv) {
-        new MyConcreteClass().instanceMain(argv);
-    }
-
-
+ *       new MyConcreteClass().instanceMain(argv);
```

Comment:
Two things: the documentation above is now out of date (it references the old annotations).  Also it's been years since Picard required `main()` methods in it's classes, I'm assuming Barclay doesn't either?  I.e. this has been inaccurate for a long time, not just made so by this commit.